msal/src/telemetry/java/com/microsoft/identity/client/ApiEvent.java

```diff
@@ -44,7 +44,9 @@ private ApiEvent(Builder builder) {
         if (null != builder.mRequestId) {
             setProperty(EventProperty.REQUEST_ID, builder.mRequestId);
         }
-        setProperty(EventProperty.AUTHORITY_NAME, HttpEvent.sanitizeUrlForTelemetry(builder.mAuthority));
+        if (!MsalUtils.isEmpty(builder.mAuthority)) {
```

Comment:
ok, i think i know why this could happen.I would add the authority url for the API event in getUsers, we look up users by authority host, that maybe the info we want. But talk to @addev-ashish, he may have some other ideas.